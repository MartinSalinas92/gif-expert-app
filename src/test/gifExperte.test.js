import { shallow } from "enzyme"
import GifExpertApp from "../gifExpert"

describe('pruebas en el componente <GifExpertApp />', ()=>{

    test('Debe mostrarse correctamente', () => {

        const wrapper= shallow(<GifExpertApp />)

        expect(wrapper).toMatchSnapshot();
        
    });
    test('debe mostrar una lista de categorias', () => {

        const categorias=['one puch', 'dragonball'];

        const wrapper=shallow(<GifExpertApp defaultCategories={categorias}/>)

        expect(wrapper).toMatchSnapshot();
        expect(wrapper.find('GifGridd').length).toBe(categorias.length);

        
    })
    
    
});