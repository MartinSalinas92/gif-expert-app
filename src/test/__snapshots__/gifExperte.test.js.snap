// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en el componente <GifExpertApp /> Debe mostrarse correctamente 1`] = `
<Fragment>
  <h2>
    GitExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol />
</Fragment>
`;

exports[`pruebas en el componente <GifExpertApp /> debe mostrar una lista de categorias 1`] = `
<Fragment>
  <h2>
    GitExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol>
    <GifGridd
      category="one puch"
      key="one puch"
    />
    <GifGridd
      category="dragonball"
      key="dragonball"
    />
  </ol>
</Fragment>
`;
